docs/examples/tukey-outlier-detection.py

```diff
@@ -0,0 +1,47 @@
+"""
+Tukey Outlier Detection
+=======================
+
+Identifying outliers in time series using
+Tukey outlier detection.
+"""
+
+# %%
+# Identifying and removing outliers from PV sensor time series
+# data allows for more accurate data analysis.
+# In this example, we demonstrate how to use
+# :py:func:`pvanalytics.quality.outliers.tukey` to identify and filter
+# out outliers in a time series.
+# We use a normalized time series example provided by the PV Fleets Initiative.
+# This example is adapted from the DuraMAT DataHub
+# clipping data set:
+# https://datahub.duramat.org/dataset/inverter-clipping-ml-training-set-real-data
```

Comment:
Move this text to the next comment paragraph, to keep the url close to the file name.